website/docs/reference/resource-configs/grants.md

```diff
@@ -1,95 +1,69 @@
 ---
-resource_types: all
-datatype: "{<dictionary>}"
+resource_types: [models,seeds,snapshots]
+datatype: "string"
 default_value: {}
+id: "grants"
 ---
 
+<Snippet src="available-prerelease-beta-banner" />
+
+You can manage access to the datasets you're producing with dbt by using grants. To implement these permissions, define grants as resource configs on each model, seed, or snapshot. Define the default grants that apply to the entire project in your `dbt_project.yml`, and define model-specific grants within each model's SQL or YAML file.
+
+The grant resource configs enable you to apply permissions at build time to a specific set of recipients and model, seed, or snapshot. When your model, seed, or snapshot finishes building, dbt ensures that the grants on its view or table match exactly the grants you have configured.
+
+dbt always finds the most efficient approach when updating grants, which varies based on the adapter you're using, and whether dbt is replacing or updating an object that already exists. You can always check the debug logs for the full set of grant and revoke statements that dbt runs.
+
+When you have a more advanced problem to solve, you might instead use grants with hooks instead of grants as resource configs. For example, if you want to create more granular row- and column-level access, use masking policies, or apply future grants. For more information on hooks, see [Hooks & operations](/building-a-dbt-project/hooks-operations).
```

Comment:
> When you have a more advanced problem to solve, you might instead use grants with hooks instead of grants as resource configs.

I think there are two conditions for this, where we'd recommend taking matters into your own hands (= hooks):
- if you need to apply grants in a more complex or custom manner, beyond what the built-in `grants` capability can provide
- you want to take advantage of more-advanced permission capabilities offered by your data platform, for which dbt does not (yet!) offer out-of-the-box support via resource configuration

Reply:
This was capture!